draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -570,24 +639,29 @@
 
         </section>
 
+	<!-- XXX JMH - the point below about "from router-X" ignores route
+	reflectors. Is this only intended to work when received directly? -->
         <section title="BGP UPDATE with standalone Site Availability Index">
-          <t>Upon receiving a BGP update message from Router-X, containing the
+          <t>Upon receiving a BGP UPDATE message from Router-X, containing the
           Metadata Path Attribute with the Site Physical Availability Index
           Sub-TLV, the next-hop should be the loopback of Router-X. The local
-          BGP peer uses local policy to evaluate the route (prefix and path
-          attributes). When the local policy processes the Metadata Attribute
+          BGP peer uses local policy to evaluate the route (prefix and Path
+          Attributes). When the local policy processes the Metadata Attribute
           with the Site Physical availability, it will use the site
           availability index to efficiently reduce or increase the preference
           for all BGP routes with the Router-X next-hop (loopback).</t>
 
           <t>The BGP UPDATE with a standalone Site Availability Index is NOT
           intended for resolving NextHop.</t>
+	  <!-- XXX JMH - the sentence above is confusing.  Aren't the routes
+	  containing the meta data intended to be loopback routes used for
+	  resolving service prefixes?  What's intended here? -->
```

Comment:
Please ignore the proposed text above.  

Here is the rewording of the Section 4.4.2 to address your comment on Line 656 & Line 643: 

"A BGP UPDATE that includes the Site Availability Index Sub-TLV without any associated routes is referred to as a BGP UPDATE with a standalone Site Availability Index. When an ingress router receives such a BGP UPDATE containing the Metadata Path Attribute with the standalone Site Physical Availability Index Sub-TLV from Router-X or its RR with the Originator-ID set to Router-X, the ingress router SHOULD use the site availability index to efficiently adjust the preference for all BGP routes with Router-X as the next-hop (loopback)."

Therefore the BGP UPDATE with a standalone Site Availability Index is NOT intended for resolving
NextHop.

Reply:
It's not clear what's meant by "associated routes".  That's probably part of what I'm asking about in terms of clarifying the use case here.
You also seem to be trying to use originator-id as a nexthop.  There's no guarantee that the originator-id will have anything to do with the bgp speaker's addresses.

Reply:
"without the associated routes" meaning that NLRI is not the route attached to egress router. Here is the correct description: 

"A BGP UPDATE that includes the Site Availability Index Sub-TLV without specifying attached routes in the NLRI, but instead using the egress router's loopback address in the NLRI, is referred to as a standalone Site Availability Index BGP UPDATE. When an ingress router receives such a BGP UPDATE containing the Metadata Path Attribute with the standalone Site Physical Availability Index Sub-TLV from Router-X or its RR with the Originator-ID equal to Router-X, the ingress router SHOULD use the site availability index to efficiently reduce or increase the preference for all BGP routes attached to Router-X ."